AppleAlarm/ListViewController.swift

```diff
@@ -18,7 +18,7 @@ class ListViewController : UITableViewController {
     
     override func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
         let cell = tableView.dequeueReusableCellWithIdentifier("AlarmItemTableViewCell", forIndexPath: indexPath) as! AlarmItemTableViewCell
-        let item = itemsList[indexPath.row]
+        let item = alarmsManager.alarms[indexPath.row]
```

Comment:
Never mind, you can keep this change in the file and deal with any merge conflicts if they occur.  LGTM.